cli/src/main/resources/help/module.txt

```diff
@@ -39,7 +39,11 @@ DESCRIPTION
 
 ARGUMENTS
 
- --name          - (required) the name of the module to be added or removed.
+ --modules-path - (optional) The absolute file path to the server's modules 
```

Comment:
@dmlloyd I thought about recommending "--module-path" (no 's') for this, to be consistent with JBoss Modules' term, but I now think it's better that it is different. This isn't a $PATH-like path which is what the JBoss Modules concept is. This is a single filesystem path where the newly added module should be placed. WDYT?

Or, perhaps --modules-dir to be even further from the JBoss Modules term.